etherslurp/follower/follower.go

```diff
@@ -116,10 +117,15 @@ nextAttempt:
 			// either we can use the Mirroring APIs once they're ready, or use the
 			// hash chain hashes to sort it out in the wash when we construct the
 			// Map from the entries in the Log.
-			if _, err := f.tc.QueueLeaves(ctx, &trillian.QueueLeavesRequest{LogId: f.logID, Leaves: []*trillian.LogLeaf{leaf}}); err != nil {
+			r, err := f.tc.QueueLeaf(ctx, &trillian.QueueLeafRequest{LogId: f.logID, Leaf: leaf})
+			if err != nil {
 				glog.Errorf("Failed to Queue block %v: %v", nextBlock, err)
 				continue nextAttempt
 			}
+			c := codes.Code(r.QueuedLeaf.GetStatus().GetCode())
+			if c != codes.OK {
+				glog.Errorf("Leaf add failed: %s", r.QueuedLeaf.GetStatus())
+			}
```

Comment:
Do you still want the "Copied to %v" message below to print even when leaf add failed?